ginga/misc/plugins/Histogram.py

```diff
@@ -441,10 +444,26 @@ def cut_levels(self):
         try:
             loval = float(self.w.cut_low.get_text())
             hival = float(self.w.cut_high.get_text())
+        except Exception as e:
+            errmsg = 'Error cutting levels: {0}'.format(str(e))
+            self.fv.showStatus(errmsg)
+            self.logger.error(errmsg)
+            return
 
+        # This is needed to avoid UserWarning from matplotlib
+        if loval == hival:
```

Comment:
To truly get rid of the warnings, we can simply just let it happen but catch it and redirect to the logger (instead of letting it spill out to the terminal). I'll give this a try shortly.